src/resource/ResourceManager.h

```diff
@@ -57,6 +59,7 @@ class ResourceManager {
                                                                                  bool loadExact = false);
 
   private:
+    bool altAssetsEnabled = false;
```

Comment:
tidy won't like this name

```suggestion
    bool mAlternateAssetsEnabled = false;
```